### ISSUE TITLE
auto pause/stop previous player

### ISSUE THREAD

**Issue by gutierri** [icasdri/mpris2controller#8]
it seems to be interesting to automatically pause previous player as soon as a new player comes into operation.

**Comment by icasdri** [comment]
I agree, that might be interesting :). Maybe as soon as new player starts playing.

**Comment by gutierri** [comment]
There is a project¹ that works for the same purpose, but inside the 
browser box. It manages players running on instances of Firefox. By 
giving the option to subtract the previous player, pause for the current 
one to run without interference, among other things.

I believe we can implement this behavior. In addition the two worked in 
parallel, but will work perfectly.

What do you think about?

**Comment by gutierri** [comment]
[1] - https://github.com/f1u77y/web-media-controller

* link
